style.css

```diff
@@ -0,0 +1,37 @@
+#main{
+    align-items: center;
+    background-color: rgb(184, 179, 179);
+    display: flex;
+    flex-direction: column ;
+}
```

Comment:
It would be better if you give margin and padding zero at the beginning of your CSS. And also don't forget to add a box-sizing border box. It will be helpful.

Reply:
- [ ] Kindly rephrase this with giving reason like why you want a specific change as "It would be better if you give margin and padding zero at the beginning of your CSS as it will make your structure clean. Kindly also add a box-sizing border-box which will make the look and feel of the website good."

Reply:
I have updated the comment as the reviewer said to make changes.